packages/fxa-content-server/server/lib/routes/get-metrics-flow.js

```diff
@@ -12,10 +12,11 @@ const logFlowEvent = require('../flow-event').logFlowEvent;
 const logger = require('../logging/log')('server.get-metrics-flow');
 const geodbConfig = config.get('geodb');
 const geodb = require('fxa-geodb')(geodbConfig);
-const remoteAddress = require('fxa-shared/express/remote-address')(
-  config.get('clientAddressDepth')
-);
-const geolocate = require('fxa-shared/express/geo-locate')(geodb)(
+const remoteAddress =
+  require('fxa-shared/express/remote-address').remoteAddress(
+    config.get('clientAddressDepth')
```

Comment:
Q: Why `config.get('clientAddressDepth')` here, but `config.clientAddressDepth` above in packages/fxa-content-server/server/lib/logging/route_logging.js#L15?

Reply:
Ah, nevermind... Seems to be an issue of `const config = require('./configuration')` versus `const config = require('../configuration').getProperties()`.